src/components/map/sectors/MapSectorsList.vue

```diff
@@ -1,38 +1,140 @@
 <template>
-    <template v-if="!isHideAtcType('firs')">
+    <template v-if="!isHideAtcType('firs') && !store.mapSettings.vatglasses?.active">
         <map-sector
             v-for="(sector, index) in firs"
             :key="sector.fir.feature.id as string + index"
             :atc="sector.atc"
             :fir="sector.fir"
         />
     </template>
+
+    <template v-else-if="!isHideAtcType('firs') && store.mapSettings.vatglasses?.active">
+        <template
+            v-for="(countryEntries, countryId) in dataStore.vatglassesActivePositions.value"
+            :key="countryId"
+        >
+            <map-vatglasses-position
+                v-for="(position, positionId) in countryEntries"
+                :key="countryId + '-' + positionId"
+                :position="position"
+            />
+        </template>
+
+
+        <map-overlay
+            class="vatglasses-overlay"
+            :model-value="vatglassesPopupIsShown"
+            :settings="{
+                position: getCoordinates,
+                offset: [15, -15],
+            }"
+            :z-index="20"
+        >
+            <common-popup-block
```

Comment:
It would be awesome if you could make cursor pointer implementation when hovering features. In same way as it's done in other components